torchci/pages/flakytest.tsx

```diff
@@ -59,12 +57,11 @@ export default function Page() {
                   <td className={styles.table}>{test.name}</td>
                   <td className={styles.table}>{test.suite}</td>
                   <td className={styles.table}>{test.file}</td>
-                  <td className={styles.table}>{test.numGreen}</td>
-                  <td className={styles.table}>{test.numRed}</td>
                   <td className={styles.table}>
-                    {convertWorkflowIDtoURLs(test.workflowIds).join("\n")}
+                    {convertJobIDtoURLs(test.jobIds).join("\n")}
                   </td>
-                  <td className={styles.table}>{test.workflowNames.join("\n")}</td>
+                  {/* The following is the same as getWorkflowJobNames in disable.ts but I couldn't figure out how to import correctly */}
+                  <td className={styles.table}>{test.workflowNames.map((value, index) => `${value} / ${test.jobNames[index]}`)}</td>
```

Comment:
You have to split that function out into its own file since parts of disable.ts can only be run on the server.

Reply:
Also more links to deep dive more would be nice.